package.json

```diff
@@ -4,10 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-
-    "build": "browserify src/main.js > public/assets/js/bundle.js",
+    "build": "browserify src/main.js --standalone addCalendar > public/assets/js/bundle.js",
```

Comment:
What does this line actually mean? What is _addCalendar_ doing - I can't seem to find any references to it within your code?

Reply:
Oh yes that is a relic from a previous implementation and can be removed, thanks for pointing it out!